volume_linux.go

```diff
@@ -39,6 +39,9 @@ func parseVolume(out string) (int, error) {
 			!useAmixer && strings.HasPrefix(s, "Volume: 0:") {
 			volumeStr := volumePattern.FindString(s)
 			return strconv.Atoi(volumeStr[:len(volumeStr)-1])
+		} else if strings.HasPrefix(s, "Volume:") {
```

Comment:
You can just remove the ` 0:` part three lines above.

Reply:
That is correct, sorry completely missed the ||. I had been meaning to pull request since before you implemented amixer support.